### ISSUE TITLE
What means max Votes?

### ISSUE THREAD

**Issue by petrk94** [ivandiazwm/phpback#142]
Hello, I would like to know what that means to have only 20 Votes?
Why its necessary? I would like to be able to give to all requests in phpback a vote and dont want to look for the amount of remaining Votes.

How to disable this?
Thanks

**Comment by MichaelTunnell** [comment]
this is a structure used from UserVoice, The idea behind it is that they have less items they can vote on so it means they would only vote on what they deem most important.

I agree with you that limiting is actually a bad thing because it will push users away since they are limited arbitrarily in how much they can participate. I have abandoned participating in every single project that uses UserVoice after I voted for the 1 or 2 things I cared to go to in the first place and I almost never go back.

UserVoice has realized themselves that it is pretty flawed and added a way for projects to not use the limiting. https://feedback.uservoice.com/knowledgebase/articles/782139-the-benefits-of-supporters-over-voting-limits

I'd vote for not limits or at the least a way to "earn access" to leave the limited group.
